docs/_static/gallery.css

```diff
@@ -0,0 +1,195 @@
+/*
+Sphinx-Gallery has compatible CSS to fix default sphinx themes
```

Comment:
Are we using sphinx gallery anywhere? If not then we probably don't need this file.